sda/internal/database/db_functions_test.go

```diff
@@ -529,6 +529,23 @@ func (suite *DatabaseTests) TestGetCorrID_sameFilePath() {
 
 }
 
+func (suite *DatabaseTests) TestGetCorrID_wrongFilePath() {
+	db, err := NewSDAdb(suite.dbConf)
+	assert.NoError(suite.T(), err, "got (%v) when creating new connection", err)
+
+	filePath := "/testuser/file10.c4gh"
+	user := "testuser"
+
+	fileID, err := db.RegisterFile(filePath, user)
+	assert.NoError(suite.T(), err, "failed to register file in database")
+	err = db.UpdateFileEventLog(fileID, "uploaded", fileID, user, "{}", "{}")
+	assert.NoError(suite.T(), err, "failed to update satus of file in database")
+
+	corrID, err := db.GetCorrID(user, "/testuser/file20.c4gh", "")
+	assert.EqualError(suite.T(), err, "sql: no rows in result set")
```

Comment:
Maybe change to a more generic assertion, e.g. 
```go
assert.Error(suite.T(), err, "expected an error for a nonexistent file path")
```
Since otherwise, this always need to be changed in case the error message from the function `GetCorrID` is changed in the future.

Reply:
That error should be an SQL error, so unless the library changes nothings needs to be done.